File: Lista 05/exercício 08.py
#Construa um algoritmo para encontrar o maior e o menor número de uma série de números positivos fornecidos pelo usuário através do teclado. O dado finalizador é o número –1, e este não deve ser considerado.

n = 0
maior = 0
menor = 10000

while True:
    n = int(input("Digite um número: "))
    if n <= 0:
        break
    if n > maior:
        maior = n
    if n < menor:
        menor = n
print(f"Maior = {maior} \nMenor = {menor}")

File: Lista 12/exercício 08.c
//Construa um algoritmo para fazer a soma de vários valores inteiros e positivos, fornecidos pelo usuário através do teclado. O dado que finaliza a sequência de entrada é o número –1, e este não deve ser considerado.

#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main()
{
    int valor, soma;
    soma = 0;
    valor = 0;

    while (valor != -1)
    {
        printf("\nDigite um valor: \n");
        scanf("%d", &valor);
        if (valor != -1)
        {
            soma += valor;
        }
    }
    printf("\nSoma = %d", soma);

    return 0;
}